docs/fixed-responsive.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Fixed and Responsive Layouts
+description: Make your visualization responsive to the screen size
+---
+
+Gosling visualizations can be fixed or responsive to the screen size.
+
+## Fixed size
+
+A Gosling visualization can have a fixed size by setting the `width` and `height` properties. The tracks will have a fixed size regardless of the screen size.
+
+When the `width` and `height` properties are set in the root of the spec, all tracks will have this height and width unless they are overridden in the track specification.
+
+```javascript
+{
+  // highlight-start
+  width: 800,
+  height: 200,
+  // highlight-end
+  // ... rest of the Gosling spec
+}
+```
```

Comment:
Could we move this section down and bring the "Responsive size" up? I think it will better reflect the general interest of users.

Reply:
Done!